novawallet/Modules/NetworkDetails/NetworkDetailsViewController.swift

```diff
@@ -0,0 +1,209 @@
+import UIKit
+
+final class NetworkDetailsViewController: UIViewController, ViewHolder {
+    typealias RootViewType = NetworkDetailsViewLayout
+    typealias ViewModel = NetworkDetailsViewLayout.Model
+
+    let presenter: NetworkDetailsPresenterProtocol
+
+    private var viewModel: ViewModel?
+    private var nodesViewModels: [RootViewType.NodeModel] = []
+
+    init(presenter: NetworkDetailsPresenterProtocol) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        view = NetworkDetailsViewLayout()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setup()
+        presenter.setup()
+    }
+}
+
+// MARK: NetworkDetailsViewProtocol
+
+extension NetworkDetailsViewController: NetworkDetailsViewProtocol {
+    func updateNodes(with viewModel: NetworkDetailsViewLayout.Section) {
+        viewModel
+            .rows
+            .forEach { row in
+                guard case let .node(nodeModel) = row else {
+                    return
+                }
+
+                let indexPath = IndexPath(
+                    row: nodeModel.index,
+                    section: 2
```

Comment:
Let's move sections indexes to Constants for the better understanding which section is hidden behind a given index